pkg_defs/ska_sun/meta.yaml

```diff
@@ -20,16 +20,20 @@ requirements:
     - setuptools_scm
     - setuptools_scm_git_archive
     - ska_helpers
+    - testr
   run:
     - python
     - ska.quatutil
     - chandra.time
     - quaternion
     - numpy
+    - ska_helpers
+    - testr
 
 test:
   imports:
     - Ska.Sun
```

Comment:
For this, I don't think we do the conda import tests but I left it with both Ska.Sun and ska_sun here.

Reply:
well, if it is not used, then why leave it there?

Reply:
Fair point.  Is it used?  It used to be that the simple "import test" was part of the conda build but I only saw it when building packages locally/manually.  For the current builds are we actually skipping it or am I just not seeing it?

There could be some reason to keep it around for the specific case of local build testing and to somewhat "announce" that this package provides both imports.